src/shibuya.js

```diff
@@ -64,7 +66,7 @@ const scheduleTask = async ({
     const xcmpExtrinsic = shibuyaHelper.createTransactExtrinsic({
         targetParaId: TuringDev.paraId,
         encodedCall: encodedTaskViaProxy,
-        proxyAccount: proxyOnTuring,
+        proxyAccount: keyring.decodeAddress(proxyOnTuring),
```

Comment:
`keyring.decodeAddress(proxyOnTuring)` returns accountId, which is needed by this call.

The rest of time proxy address needs to be encoded and looks like a Shibuya or Turing address, such as `aNFdmcr5ERyubSoB6mqb2q4qYKT8pPTLnFe8Q2oGeKj7Sze`